### PR TITLE
added Copyright queston to commit checklist

### PR DESCRIPTION
**Pull request by sguermond**
Fixed issue #937 

- [x] Is it mergeable?
- [x] Did it pass the tests?
- [x] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [x] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [x] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [x] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?

### CODE REVIEW

**Comment by standage** [comment]
Ok to test, Jenkins.

**Comment by mr-c** [comment]
Jenkins, ok to test

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,8 @@
+2015-04-14  Sarah Guermond <sarah.guermond@gmail.com>
```
Comment (mr-c) [inline ChangeLog]: Two spaces between your name and your bracketed email address, please.

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+
+   * khmer/doc/dev/coding-guidelines-and-review.rst: added copyright question
```
Comment (mr-c) [inline ChangeLog]: No leading 'khmer' needed.

**Comment by mr-c** [comment]
Otherwise this looks good to me, thanks!

**Comment by sguermond** [comment]
@mr-c Ready for review!

**Comment by mr-c** [comment]
@sguermond Congratulations on your first commit to the khmer project! Your name will be included in the release notes for the next version and you'll be listed amongst our other contributors in the next software release paper.

**Pull request merged**